blocks/form/form.js

```diff
@@ -8,6 +8,10 @@ import decorateUTM from './utm.js';
 const SITE_KEY = '6Lc0qYoiAAAAADNOimWbvoSdn2YawNsa6Wbxejpp';
 const FORM_SUBMIT_ENDPOINT = 'https://franklin-submit-wrapper.mammotome.workers.dev';
 
+function getLocaleRegion() {
+  return (getMetadata('locale') || 'en-US').split('-');
```

Comment:
Before it was en and now default is en-US. Is it expected change?

Reply:
The default value in metadata sheet has been changed to `en-US` that why I updated the value.